### PR TITLE
deps: Bump Three.js to r151

### LINKED ISSUE #1198: `uv2_vertex` and `uv2_pars_vertex` failing mtoon VRM rendering in Three v151
In Three.js v151 there were changes that removed `uv2_vertex` and `uv2_pars_vertex` which cause issues loading VRM files that have the mtoon shader.

Here is the Three.js change: https://github.com/mrdoob/three.js/commit/5c5a5c74c6611d3f14dd3e58c5139afd3e4ea182#diff-c2bc4656a48180f106fc5f8813c5d6fd1fe51f8af74b7ac5e1939661379d4841

I found that mtoon uses `uv2_vertex` and `uv2_pars_vertex` in Line 42 and 17 of the mtoon.vert file. 
https://github.com/pixiv/three-vrm/blob/e4a2dcc7de2e0f5801e91afe9b4150982a0b06c4/packages/three-vrm-materials-mtoon/src/shaders/mtoon.vert#L42

I attempted to comment them out, but as expected it broke things further. Reporting here as I try to solve the problem. :)

### PR DESCRIPTION
**Pull request by 0b5vr**
will resolve #1198 

### Description

This PR bumps Three.js to r151.

This PR also fixes shader errors in r151 while maintaining compatibility before r151.

Also added a feature test scene for MToon.

See: https://github.com/mrdoob/three.js/releases/tag/r151

### TODO

- [x] Make sure the shader is working on r151 and r150

### CODE REVIEW

File: packages/three-vrm-materials-mtoon/src/shaders/mtoon.frag [hunk 1]
```diff
@@ -161,7 +171,7 @@ vec3 getDiffuse(
 }
 
 void RE_Direct_MToon( const in IncidentLight directLight, const in GeometricContext geometry, const in MToonMaterial material, const in float shadow, inout ReflectedLight reflectedLight ) {
-  float dotNL = saturate( dot( geometry.normal, directLight.direction ) );
+  float dotNL = clamp( dot( geometry.normal, directLight.direction ), -1.0, 1.0 );
```
Comment (0b5vr) [inline packages/three-vrm-materials-mtoon/src/shaders/mtoon.frag:174]: 光の当たっていない側にトゥーンのfeatherを動かせられなかったのを修正しました。

**Comment by ke456-png** [comment]
👍 
image
examples/feature-test.html

File: packages/three-vrm-materials-mtoon/src/shaders/mtoon.vert [hunk 2]
```diff
@@ -36,10 +49,22 @@ void main() {
 
   // #include <uv_vertex>
   #ifdef MTOON_USE_UV
-    vUv = ( uvTransform * vec3( uv, 1 ) ).xy;
+    // COMPAT: pre-r151 uses a common uvTransform
+    #if THREE_VRM_THREE_REVISION >= 151
+      vUv = uv;
+    #else
+      vUv = ( uvTransform * vec3( uv, 1 ) ).xy;
```
Comment (ke456-png) [inline packages/three-vrm-materials-mtoon/src/shaders/mtoon.vert:56]: [memo]

**Review by ke456-png — APPROVED** [review]
NICE 👍

**Review by ke456-png — APPROVED** [review]
NICE 👍

**Pull request merged**